kratos/python/add_response_functions_to_python.cpp

```diff
@@ -55,7 +55,18 @@ void AddResponseFunctionsToPython(pybind11::module& m)
         .def("Initialize", &AdjointResponseFunction::Initialize)
         .def("InitializeSolutionStep", &AdjointResponseFunction::InitializeSolutionStep)
         .def("FinalizeSolutionStep", &AdjointResponseFunction::FinalizeSolutionStep)
-        .def("CalculateValue", &AdjointResponseFunction::CalculateValue);
+        .def("CalculateValue", &AdjointResponseFunction::CalculateValue)
+        .def("CalculateGradient", static_cast<void(AdjointResponseFunction::*)(const Condition&, const Matrix&, Vector&, const ProcessInfo&)>(&AdjointResponseFunction::CalculateGradient))
```

Comment:
shouldn't this also work without the `static_cast`?

Reply:
It is an overloaded method for `Element` and `Condition`, i'm not sure how to distinguish between them if casting is not used.

Reply:
ok, also not sure here
In any case I think this can all be removed with `pybind::overload_cast` when we have C++14